Games/types/Mafia/roles/cards/DiesWithVillageCondemn.js

```diff
@@ -0,0 +1,24 @@
+const Card = require("../../Card");
+const Action = require("../../Action");
+
+module.exports = class DiesWithVillageCondemn extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.listeners = {
+            "meetingFinish": function(meeting) {

```

Comment:
i will change this to work more like gallis (queue an action), so we can account for immunities